.github/workflows/build.yml

```diff
@@ -20,15 +20,11 @@ jobs:
       DB_PASSWORD: postgres
```

Comment:
Is this useful here? terraform doesn't use it and docker-compose defines it internally

Reply:
yep....it is being used and it is referrenced in the database.yml file.